src/main.py

```diff
@@ -1 +1,87 @@
-# Your code lives here, Good luck!
\ No newline at end of file
+### packages importing
+import os
+import cv2
+import numpy as np
+import json
+import jiwer
+import pytesseract
+import time
+
+
+### input images reading
+def read_img_from_source():
+  images_array = []
+  absolute_input_path = os.path.abspath("../input")
+  input_images_nouns = os.listdir(absolute_input_path)
+  input_images_nouns.remove(".ipynb_checkpoints")
+  for input_images_noun in input_images_nouns:
+     img = cv2.imread(absolute_input_path+"/"+input_images_noun, cv2.IMREAD_COLOR)
+     img = cv2.cvtColor(img,cv2.COLOR_BGR2RGB)
+     images_array = images_array + [img]
+  return images_array,input_images_nouns
+
+### apply preprocessing operations on brute images
+def preprocess_images(images_array):
+  processed_arrays = []
+  for img in images_array:
+    # resizing image to (500,500)
+    resized_img = cv2.resize(img,(500,500), interpolation = cv2.INTER_AREA)
+    norm_img = np.zeros((resized_img.shape[0], resized_img.shape[1]))
+    # normalizing pixels value between 0 and 1
+    normalized_img = cv2.normalize(resized_img, norm_img, 0, 255, cv2.NORM_MINMAX)
+    processed_arrays += [normalized_img]
+  return processed_arrays
+
+
+### pytesseract image text conversion
+def extract_text_content(processed_arrays):
+  images_predicted_text_content = []
+  for processed_img in processed_arrays:
+    images_predicted_text_content += [pytesseract.image_to_string(processed_img)]
+  return images_predicted_text_content
+
+### post-process output text produced by pytesseract
+def text_post_processing(images_predicted_text_content,input_files_names):

```

Comment:
A bit light in terms of postprocessing, since the results will be highly noisy with these types of images